webpack/webpack5-loader-load-themed-styles/src/LoadThemedStylesLoader.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+/**
+ * This simple loader wraps the loading of CSS in script equivalent to
+ *  require("load-themed-styles").loadStyles('... css text ...').
+ * @packageDocumentation
+ */
+
+import type { LoaderContext } from 'webpack';
```

Comment:
```suggestion
import type {
  LoaderContext,
  LoaderModule
} from 'webpack';
```

Reply:
I directly imported the `PitchLoader` type instead of the LoaderModule type since just the pitch function is being exported.